### PR TITLE
bower.json and prototype of conversation between person completed

### PR DESCRIPTION
**Pull request by Earthii**
To test: 

Go to terminal and do ==> node index
2) then open two different tab (representing two people)
3) chat between the two tabs.
4) observe the live chat room

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Socket.IO chat</title>
+        <style>
```
Comment (almiche) [inline index.html]: Can you put all the styling tags in a css file ?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            form { background: #000; padding: 3px; position: fixed; bottom: 0; width: 100%; }
+            form input { border: 0; padding: 10px; width: 90%; margin-right: .5%; }
+            form button { width: 9%; background: rgb(130, 224, 255); border: none; padding: 10px; }
+            #messages { list-style-type: none; margin: 0; padding: 0; }
+            #messages li { padding: 5px 10px; }
+            #messages li:nth-child(odd) { background: #eee; }
+        </style>
+        <script src="/socket.io/socket.io.js"></script>
+        <script src="/bower_components/jquery/dist/jquery.min.js"></script>
+        <script>
```
Comment (almiche) [inline index.html]: I'm guessing this will go in a separate .js file later and is here only for testing ?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                $('form').submit(function(){
+                    var message = $('#m');
+                    socket.emit('chat message', message.val());
+                    message.val('');
+                    return false;
+                });
+            });
+            //when the client receives an emit (new message) from server
+            socket.on('chat message', function(msg){
+                $('#messages').append($('<li>').text(msg))
```
Comment (almiche) [inline index.html]: I'm guessing this will go in a separate .js file later and is here only for testing ?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        <title>Socket.IO chat</title>
+        <style>
+            * { margin: 0; padding: 0; box-sizing: border-box; }
+            body { font: 13px Helvetica, Arial; }
+            form { background: #000; padding: 3px; position: fixed; bottom: 0; width: 100%; }
+            form input { border: 0; padding: 10px; width: 90%; margin-right: .5%; }
+            form button { width: 9%; background: rgb(130, 224, 255); border: none; padding: 10px; }
+            #messages { list-style-type: none; margin: 0; padding: 0; }
+            #messages li { padding: 5px 10px; }
+            #messages li:nth-child(odd) { background: #eee; }
```
Comment (almiche) [inline index.html]: Can you put all the styling tags in a css file ?

File: index.html (hunk 3 above)
Comment (Earthii) [inline index.html]: Yeah its only ehre for testing

File: index.html (hunk 4 above)
Comment (Earthii) [inline index.html]: Its a prototype remember, im not doing the front-end

File: index.js [hunk 5]
```diff
@@ -1,4 +1,34 @@
...
+    });
+
+    //when user leaves, do something
+    socket.on('disconnect', function(){
+       console.log('user disconnected')
+    });
+});
+
+http.listen(3000,function(){
+    console.log('listening on*: 3000');
```
Comment (jusleg) [inline index.js]: can you change to port 9001. This port was open to try the chat on the server

File: bower.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "soen341",
+  "description": "Soen 341 chat app",
+  "main": "index.js",
+  "authors": [
+    "Earthii <xiao_eric@hotmail.com>"
```
Comment (jusleg) [inline bower.json]: Add the whole team

File: index.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Socket.IO chat</title>
```
Comment (jusleg) [inline index.html]: Put proper title

File: index.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+    <head>
```
Comment (jusleg) [inline index.html]: Add meta charset Attribute
`<meta charset="UTF-8">`

**Comment by jusleg** [comment]
#18 #23 #24

**Pull request merged**